cve_bin_tool/nvd_api.py

```diff
@@ -21,25 +20,13 @@
 from cve_bin_tool.error_handler import ErrorMode, NVDServiceError
 from cve_bin_tool.log import LOGGER
 
-NVD_FILENAME_TEMPLATE = "nvdcve-1.1-{}.json.gz"
-DISK_LOCATION_DEFAULT = os.path.join(os.path.expanduser("~"), ".cache", "cve-bin-tool")
-
-
 FEED = "https://services.nvd.nist.gov/rest/json/cves/1.0"
+NVD_CVE_STATUS = "https://nvd.nist.gov/rest/public/dashboard/statistics"
 
 PAGESIZE = 2000
 MAX_FAIL = 5
 # Interval in seconds between successive requests
-INTERVAL_PERIOD = 2
-
-
-def filter_by_id(cve, update_data):
-    """Function to filter out duplicate CVE entries in case of incremental update"""
-    cve_id = cve["cve"]["CVE_data_meta"]["ID"]
-
-    return all(
-        cve_id != update_cve["cve"]["CVE_data_meta"]["ID"] for update_cve in update_data
-    )
+INTERVAL_PERIOD = 3
 
 
 class NVD_API:
```

Comment:
Just a style nit but class names should normally use the CapWords convention (same for `TestNVD_API`).

Reply:
Thanks @Molkree 
I will keep that in mind.

Reply:
According to this [style guide](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles), it is better to use all caps while using acronyms. As `HTTPServerError` is better than `HttpServerError`.
We are using the same convention for CVEScanner and CVEDB.

Reply:
> We are using the same convention for CVEScanner and CVEDB.

`CVEScanner` and `CVEDB` follow this convention, yes 😃 
But `NVD_API` is not because it has an underscore. It's actually pretty similar to `CVEDB` (vs `CVE_DB`).

[Class Names](https://www.python.org/dev/peps/pep-0008/#class-names)

Reply:
I'm willing to accept NVD_API for readability reasons here, regardless of conventions.  Honestly, I'd be willing to rework CVEDB to be CVE_DB for the same reason, but I suspect that veers well into waste-of-time territory.